source/supported-hardware/apollo-lake-crb.rst

```diff
@@ -55,47 +30,72 @@ Stitching
   This image contains additional firmware ingredients that are required to boot on |APL|.
 
 .. note::
-.... ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**. 
-.... Unzip the firmware images that contains two |APL| firmware images, one is Debug version and one is Release version, both of them can be used for stitch SBL IFWI. 
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**. 
+  Unzip the firmware images that contains two |APL| firmware images, one is Debug version and one is Release version, both of them can be used for stitch SBL IFWI. 
 
 
 2. Stitch |SPN| images into downloaded BIOS image::
 
-    python Platform/ApollolakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE> -s Outputs/apl/Stitch_Components.zip -o <SBL_IFWI_IMAGE>
+    python Platform/ApollolakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE> -s Outputs/apl/Stitch_Components.zip -o <SBL_IFWI_IMAGE> 

```

Comment:
Please mention -i means "input" and -o means "output"

Reply:
Will add it in the next PR.